src/components/Steps/__stories__/steps.stories.mdx

```diff
@@ -0,0 +1,143 @@
+import Steps from "../Steps";
+import { useMemo } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import Icon from "../../Icon/Icon.jsx"
+import { NavigationChevronLeft, NavigationChevronRight } from "../../Icon/Icons"
+import Tipseen from "../../Tipseen/Tipseen.jsx"
+import TipseenWizard from "../../Tipseen/TipseenWizard";
+import { modifiers } from "./helpers.js";
+import { BREADCRUBMS, TAB, WIZARD } from "../../../storybook/components/related-components/component-description-map";
+import "./steps.stories.scss";
+
+<Meta
+  title="Components/Steps"
+  component={ Steps }
+/>
+
+<!--- Component template -->
+
+export const stepsTemplate = createComponentTemplate(Steps);
+  const steps5 = [<div></div>, <div></div>, <div></div>, <div></div>, <div></div>];
+  const steps6 = [<div></div>, <div></div>, <div></div>, <div></div>, <div></div>, <div></div>];
+
+<!--- Component documentation -->
+
+# Steps
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Steppers display progress through a sequence of logical and numbered steps. They may also be used for navigation.
+<Canvas>
+  <Story name="Overview"
+         args={ {steps: steps5, activeStepIndex: 2} }>
+    { stepsTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Steps } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Stepper must have three or more steps.",
+  "The stepper shows users where they are in the process, and can be used to navigate through the process by selecting steps.",
+  "If a task needs more than six steps, consider simplifying the process or breaking it up into multiple tasks.",
+]}/>
+
+## Variants
+### Types
+Steps with a number view or gallery view.
+<Canvas>
+  <Story name="Types">
+    {() => {
+      const steps5 = useMemo(() =>  [<div></div>, <div></div>, <div></div>, <div></div>, <div></div>], [] );
+      const steps6 = useMemo(() =>  [<div></div>, <div></div>, <div></div>, <div></div>, <div></div>, <div></div>], [] );
+      return(
+        <div className="monday-style-story-steps_column-wrapper">
+          <Steps type={Steps.types.NUMBERS} steps={steps6} activeStepIndex={4} />
+          <Steps steps={steps5} activeStepIndex={2} />
+          <Steps steps={steps5} activeStepIndex={2} areNavigationButtonsHidden className="monday-style-story-steps_padding" />
+        </div>
+      )
+    }};
+  </Story>
+</Canvas>
+
+### On primary
+<Canvas>
+  <Story name="On primary">
+    {() => {
+      const steps5 = useMemo(() =>  [<div></div>, <div></div>, <div></div>, <div></div>, <div></div>], [] );
```

Comment:
please rename steps5 variable to steps and remove the steps6 variable which not in use